projects/dsp-ui/src/lib/viewer/services/value-operation-event.service.ts

```diff
@@ -1,5 +1,6 @@
 import { Subject, Subscription } from 'rxjs';
 import { filter, map } from 'rxjs/operators';
+import { ReadValue } from '@dasch-swiss/dsp-js';
```

Comment:
VSC warning: Import sources within a group must be alphabetized.

Reply:
fixed in 97213e5